src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -1,19 +1,10 @@
-import { MethodHandle, System_Object, System_String, System_Array, Pointer, Platform } from '../Platform';
+import { System_Object, System_String, System_Array, Pointer, Platform } from '../Platform';
 import { getFileNameFromUrl } from '../Url';
 import { attachDebuggerHotkey, hasDebuggingEnabled } from './MonoDebugger';
 import { showErrorNotification } from '../../BootErrors';
 
-const assemblyHandleCache: { [assemblyName: string]: number } = {};
-const typeHandleCache: { [fullyQualifiedTypeName: string]: number } = {};
-const methodHandleCache: { [fullyQualifiedMethodName: string]: MethodHandle } = {};
-
-let assembly_load: (assemblyName: string) => number;
-let find_class: (assemblyHandle: number, namespace: string, className: string) => number;
-let find_method: (typeHandle: number, methodName: string, unknownArg: number) => MethodHandle;
-let invoke_method: (method: MethodHandle, target: System_Object, argsArrayPtr: number, exceptionFlagIntPtr: number) => System_Object;
-let mono_string_array_new: (length: number) => System_Array<System_String>;
 let mono_string_get_utf8: (managedString: System_String) => Mono.Utf8Ptr;
-let mono_string: (jsString: string) => System_String;
+let mono_bind_static_method: (fqn: string) => (args: (System_Object | null)[]) => System_Object;
```

Comment:
I don't think it's necessary to declare this as a separate variable, since it already exists as `Module.mono_bind_static_method`.

Instead, consider editing `MonoType.d.ts` (in the same dir as this file) and add an entry inside the definition of `Module` to match this function. Then it will be possible to call `Module.mono_bind_static_method` directly and in a strongly-typed way anywhere else.

Reply:
Oh wait I see you already did add that into `MonoTypes.d.ts`. So you could just use that directly and don't have to create another variable referencing it here.

Did you do this for perf? If so I suspect it's not worth it. We normally cache the result of calling `mono_bind_static_method` so we don't need an extra layer of caching for just the evaluation of `Module.mono_bind_static_method`.